integration_test/README.md

```diff
@@ -25,7 +25,8 @@ the test will:
     `applications/<application>/<platform>/post` on the VM
 1.  Configure the the Ops Agent to look for the application's metrics by
     running
-    `agent/ops-agent/<platform>/enable_<application>` on the VM
+    `agent/ops-agent/<platform>/enable_<application>` on the VM. This might
+    be a no-op for some applications.
```

Comment:
Ah, `This might be a no-op for some applications.` should be following the previous line, right? aka `applications/<application>/<platform>/post`might be no-op for applications like Nginx.

Reply:
done